templates/artist/artist_detail.html

```diff
@@ -199,10 +228,56 @@ <h1 class="header-font">{{ artist.name }}</h1>
 
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
     <script type="text/javascript">
         var perdiem_fee = {{ PERDIEM_FEE }};
         var artist_name = '{{ artist.name|escapejs }}';
         {% if campaign %}
```

Comment:
Can you add a newline here before `{% if campaign %}` to separate those two parts?